src/translator_ingest/ingests/bgee/bgee_rig.yaml

```diff
@@ -0,0 +1,192 @@
+      - file_name:  Gallus_gallus_expr_simple.tsv.gz
+        included_records:  
+        fields_used: GeneID, Genename, AnatomicalentityID, Anatomicalentityname, Expression, Callquality, FDR, Expressionscore, Expressionrank
+
+    filtered_content:  # (optional, multivalued, range = FilteredContent) Records from relevant files that are not included in the ingest, and why.
+      - file_name:  Homo_sapiens_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Rattus_norvegicus_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Mus_musculus_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Danio_rerio_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Xenopus_laevis_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Drosophila_melanogaster_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Caenorhabditis_elegans_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Canis_lupus_familiaris_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Bos_taurus_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Sus_scrofa_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\""
+        rationale:  
+      - file_name:  Gallus_gallus_expr_simple.tsv.gz
+        filtered_records: "'Expression rank' >= 10,000 OR 'Expression score' <= 70 OR 'FDR' >= 0.05 OR 'Expression' == \"absent\"" 
+        rationale:  
+
+    future_considerations:  # (optional, multivalued, range = FutureContentConsiderations) Content additions/changes to consider for future iterations.
+      - category:  edge_content
+        consideration:  Filter some organisms which are already filtered by Monarch.# (required, range = string)
```

Comment:
if you can update some of this to be translator specific that would be very helpful for next round.